pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -162,7 +168,44 @@ func (pl *CSILimits) filterAttachableVolumes(
 			pvcName = ephemeral.VolumeClaimName(pod, &vol)
 			isEphemeral = true
 		default:
+			// Inline Volume does not have PVC.
+			// Need to check if CSI migration is enabled for inline volume.
+			isMigratable = pl.translator.IsInlineMigratable(&vol)
+			// If the volume is not migratable, it will be count in non_csi filter.
+			if !isMigratable {
+				continue
+			}
+		}
+
+		if isMigratable {
+			// Check if the intree provisioner CSI migration has been enabled.
+			inTreeProvisionerName, err := pl.translator.GetInTreePluginNameFromSpec(nil, &vol)
+			if err != nil {
+				return fmt.Errorf("error looking up provisioner name for volume: %v", vol)
```

Comment:
Shall we add the `err` in the error message?

Reply:
Done